src/zenml/models/step_run_models.py

```diff
@@ -82,6 +83,7 @@ class StepRunResponseModel(StepRunBaseModel, WorkspaceScopedResponseModel):
         default={},
         title="Metadata associated with this step run.",
     )
+    step_logs: Optional["LogsResponseModel"] = None
```

Comment:
Just logs seems more appropriate IMO, we're already in the `StepRunModel`.

Reply:
I'm wondering if it wouldn't be easier to just have a `logfile_uri` here instead? Is there anything we gain from having the separate table? The connection to the artifact store already exists through the stack associated with the pipeline run.

Reply:
@schustmi @fa9r and I discussed it and I wanted it to be a different table because:

- The stack could be updated leaving this step dangling.
- In the future, we might want to add multiple log files or add fancy magic like a logdriver

Reply:
Ah the stack updated but the artifact store still exists? I guess yes in that case it might help 👍